README.md

```diff
@@ -52,6 +52,15 @@ page.posts
 # => [#<Fb::Post: id="1234", type="video">, #<Fb::Post: id="5678", type="video">]
 ```
 
+Pass `since` (`Time`) and `until` (`Time`) options to get posts in between the two times.
+
+```ruby
+page.like_count until: Date.today - 7
+options = { since: Time.parse((Date.today - 7).to_s), until: Time.parse(Date.today.to_s) }
```

Comment:
@S1v4 This example is a little complex. How about something on the line of `Time.now`?